projects/steering/src/steering_can.c

```diff
@@ -7,16 +8,18 @@
 #include "steering_digital_input.h"
 #include "steering_events.h"
 
+static bool turning = false;
+
 StatusCode steering_can_process_event(Event *e) {
   switch (e->id) {
     case STEERING_INPUT_HORN_EVENT:
       CAN_TRANSMIT_HORN((EEHornState)e->data);
       break;
     case STEERING_DRL_1_EVENT:
-      CAN_TRANSMIT_LIGHTS(EE_LIGHT_TYPE_DRL, (EELightState)e->data);
+      if (turning == false) CAN_TRANSMIT_LIGHTS(EE_LIGHT_TYPE_DRL, EE_LIGHT_STATE_ON);

```

Comment:
I think the idea is we want to disable left/right signals when one DRL input is on, rather than disabling DRL inputs when left or right is on. (Also you can just do `!turning` instead of `turning == false`.)

Reply:
Oh my bad, I must have read that backwards

Reply:
Actually, I talked to @aashmikamali , and she said it's the opposite. It's if left/right signal is on, then we disable DRL.

Reply:
Ah okay